Core/GDCore/IDE/CaptureOptions.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+#include <memory>
+#include <vector>
+
+#include "GDCore/String.h"
+
+namespace gd {
+
+class GD_CORE_API Screenshot {
+ public:
+  Screenshot();
+  virtual ~Screenshot() {};
+
+  void SetTiming(int timing_) { timing = timing_; }
+  const long GetTiming() const { return timing; }
+
+  void SetSignedUrl(const gd::String& signedUrl_) { signedUrl = signedUrl_; }
+  const gd::String& GetSignedUrl() const { return signedUrl; }
+
+  void SetPublicUrl(const gd::String& publicUrl_) { publicUrl = publicUrl_; }
+  const gd::String& GetPublicUrl() const { return publicUrl; }
+
+ private:
+  int timing;
```

Comment:
```suggestion
  int timing = 0;
```

 Otherwise this has a risk to be initialized to anything that lives in memory here (clang-tidy would probably catch this)